signing_clients/apps.py

```diff
@@ -324,18 +322,20 @@ def signature(self):
         # section signatures
         return self.signatures.header + b"\n"
 
-    def make_signed(self, signature, outpath=None, sigpath=None):
-        outpath = outpath or self.outpath
+    def make_signed(self, signed_manifest, signature, outpath, sigpath):
         if not outpath:
             raise IOError("No output file specified")
```

Comment:
I thought so too, but wasn't sure if we were maybe protecting against outpath='' or something wacky like that.